science/gmtsar-data/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                gmtsar-data
+version             1.0
+categories          science
+platforms           darwin
+license             GPL-3
```

Comment:
please set the ```license``` to ```Restrictive Nomirror```, this will avoid mirroring the distfile and avoid storing archives; it will just always be downloaded from ftp://topex.ucsd.edu/pub/gmtsar/. See the [mailinglist](https://lists.macports.org/pipermail/macports-dev/2019-March/040394.html) for more information.